exts/cesium.powertools/cesium/powertools/ui/georef_helper_window.py

```diff
@@ -0,0 +1,149 @@
+            self._wgs84_latitude_model.get_value_as_float()
+        )
+        cesium_prim.GetAttribute("cesium:georeferenceOrigin:longitude").Set(
+            self._wgs84_longitude_model.get_value_as_float()
+        )
+        cesium_prim.GetAttribute("cesium:georeferenceOrigin:height").Set(
+            self._wgs84_height_model.get_value_as_float()
+        )
+
+    def _build_fn(self):
+        """Builds out the UI buttons and their handlers."""
+
+        with ui.VStack(spacing=4):
+            label_style = {"Label": {"font_size": 16}}
+
+            ui.Label(
+                "Enter coordinates in any EPSG CRS to convert them to ECEF and WGS84",
+                word_wrap=True,
+                style=label_style,
+            )
+            ui.Spacer(height=10)
+            ui.Label("Your Project Details:", height=20)
+
+            # TODO: Precision issues to resolve
+            def on_coordinate_update(event):
+                self.convert_coordinates()
+
+            # Define the SimpleValueModels for the UI
+            self._epsg_model = ui.SimpleIntModel(28356)
+            self._epsg_name_model = ui.SimpleStringModel("")
+            self._easting_model = ui.SimpleFloatModel(503000.0)
+            self._northing_model = ui.SimpleFloatModel(6950000.0)
+            self._elevation_model = ui.SimpleFloatModel(0.0)
+
+            # Add the value changed callbacks
+            self._epsg_model.add_value_changed_fn(on_coordinate_update)
+            self._easting_model.add_value_changed_fn(on_coordinate_update)
+            self._northing_model.add_value_changed_fn(on_coordinate_update)
+            self._elevation_model.add_value_changed_fn(on_coordinate_update)
+
+            # TODO: Make EPSG an autocomplete field
+
+            int_field_with_label("EPSG Code", model=self._epsg_model)
+            string_field_with_label("EPSG Name", model=self._epsg_name_model, enabled=False)
+            float_field_with_label("Easting / X", model=self._easting_model)
+            float_field_with_label("Northing / Y", model=self._northing_model)
+            float_field_with_label("Elevation / Z", model=self._elevation_model)
+
+            ui.Spacer(height=10)
+
+            # TODO: It would be nice to be able to copy these fields, or potentially have two way editing
+
+            ui.Label("WGS84 Results:", height=20)
+            self._wgs84_latitude_model = ui.SimpleFloatModel(0.0)
+            self._wgs84_longitude_model = ui.SimpleFloatModel(0.0)
+            self._wgs84_height_model = ui.SimpleFloatModel(0.0)
+            float_field_with_label("Latitude", model=self._wgs84_latitude_model, enabled=False)
+            float_field_with_label("Longitude", model=self._wgs84_longitude_model, enabled=False)
+            float_field_with_label("Elevation", model=self._wgs84_height_model, enabled=False)
+
+            ui.Spacer(height=10)
+
+            ui.Label("ECEF Results:", height=20)
```

Comment:
I'd suggest setting `height=0` on these and instead using a style that contains increases the font size and applies bold on it.

Reply:
Done - though I couldn't see a bold specific style setting.  There is an option to specify different TTF's, wasn't sure if there were fonts built in or I needed to supply, in the end for the sake of time I just increased the heading font size slightly which works ok.